rpm/0007-Add-RFC7636-aka-PKCE-support.patch

```diff
@@ -22,18 +22,19 @@ Linux but might behave differently on other platforms and is not
 portable to non Unix-like OSs.
 
 Signed-off-by: Tomi Leppänen <tomi.leppanen@jolla.com>
+Signed-off-by: Björn Bidar <bjorn.bidar@jolla.com>
 ---
  src/oauth2data.h     | 10 ++++++++++
  src/oauth2plugin.cpp | 44 ++++++++++++++++++++++++++++++++++++++++++++
  2 files changed, 54 insertions(+)
 
 diff --git a/src/oauth2data.h b/src/oauth2data.h
-index 31184974f2cd25be7df5483415496a13f9df2cf0..7b4623b21a6678f7d32b4d4b1e8915191a058c29 100644
+index dacd3d499cd19ff9345e1e3161bdd8d08a80a82d..15097de69a79e1ccfa72c68e26a6ea32e07114c1 100644
 --- a/src/oauth2data.h
 +++ b/src/oauth2data.h
-@@ -125,6 +125,16 @@ namespace OAuth2PluginNS {
+@@ -120,6 +120,16 @@ namespace OAuth2PluginNS {
           */
-         SIGNON_SESSION_DECLARE_PROPERTY(QString, TokenQuery);
+         SIGNON_SESSION_DECLARE_PROPERTY(QString, AuthQuery);
  
```

Comment:
Without this change the patches don't apply using git am, the change is from git format-patch not
 from me.